src/eradiate/scenes/bsdfs/_hapke.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+import attrs
+import mitsuba as mi
+import pint
+import pinttr
+
+from ._core import BSDF
+from ..core import traverse
+from ..spectra import Spectrum, spectrum_factory
+from ... import validators
+from ...attrs import documented, parse_docs
+from ...kernel import TypeIdLookupStrategy, UpdateParameter
+from ...units import unit_context_config as ucc
+
+
+@parse_docs
+@attrs.define(eq=False, slots=False)
+class HapkeBSDF(BSDF):
+    """
+    Hapke BSDF [``rtls``].
```

Comment:
```suggestion
    Hapke BSDF [``hapke``].
```
You're also missing the factory registration (in `_core`).